app/src/sharedTest/java/org/oppia/android/app/options/OptionsFragmentTest.kt

```diff
@@ -85,6 +85,17 @@ import org.robolectric.annotation.LooperMode
 import javax.inject.Inject
 import javax.inject.Singleton
 
+private const val KEY_READING_TEXT_SIZE_PREFERENCE_TITLE = "READING_TEXT_SIZE_PREFERENCE"
+private const val APP_LANGUAGE_PREFERENCE_TITLE_EXTRA_KEY =
+  "AppLanguageActivity.app_language_preference_title"
+private const val KEY_AUDIO_LANGUAGE_PREFERENCE_TITLE = "AUDIO_LANGUAGE_PREFERENCE"
+private const val KEY_READING_TEXT_SIZE_PREFERENCE_SUMMARY_VALUE =
+  "READING_TEXT_SIZE_PREFERENCE_SUMMARY_VALUE"
+private const val APP_LANGUAGE_PREFERENCE_SUMMARY_VALUE_EXTRA_KEY =
+  "AppLanguageActivity.app_language_preference_summary_value"
+private const val KEY_AUDIO_LANGUAGE_PREFERENCE_SUMMARY_VALUE =
+  "AUDIO_LANGUAGE_PREFERENCE_SUMMARY_VALUE"
```

Comment:
@anandwana001 and @Arnold2381 This passes the tests but thats no the correct way to write tests. In future if we change these constants in production files then these tests will again start failing and we will need to fix that too. 

@anandwana001 WDYT?

Reply:
@rt4914 I tried importing the constants but the bazel tests are failing.

Reply:
@rt4914 currently, we don't directly import the variables, but we use the get function from the test file to get the original key in the production file.  

Reference - https://github.com/oppia/oppia-android/blob/82d8ee4c3cd4c3a6b90703c805b48e8bdd99db38/app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivity.kt#L66

Reply:
Actually we need to make all of these keys consistent by defining them in `companion object` and then using it wherever needed.

Reply:
For this PR we can use any solution but the main point is to use same const in production and in test file.

Reply:
yeah, that's why we can use the get function. 
Bazel doesn't support importing.

Reply:
Okay than we will need to use `function` for all intent values. Sure, lets do it for this PR and than discuss this idea with Jan and Ben too.

Reply:
@rt4914 is it okay now? PTAL